v/vip/vip-v8.0.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'vip'
```

Comment:
VIP has a dependency on Java. Could you add an EasyBuild dependency on https://github.com/molgenis/take-it-easyconfigs/blob/main/j/Java/Java-21-LTS.eb?

Side notes:
1. I think https://github.com/molgenis/take-it-easyconfigs/blob/main/j/Java/Java-21-LTS.eb is broken because 21.0.2 does not exist in https://github.com/molgenis/take-it-easyconfigs/tree/main/j/Java (but is available on for example gearshift)
2. Java 21.0,5 is available, might be a good time to update Java-21-LTS to point to 21.0,5 instead of 21.0.2